fedbadges/rules.py

```diff
@@ -426,7 +425,7 @@ def matches(self, msg):
         )
 
         # Force lowercase, https://github.com/fedora-infra/tahrir/issues/315
-        actual_packages = set(map(unicode.lower, actual_packages))
+        actual_packages = set(map(str.lower, actual_packages))
```

Comment:
I'd use a set comprehension on python 3 instead.

```suggestion
        actual_packages = {p.lower() for p in actual_packages}
```